lib/linguist/languages.yml

```diff
@@ -8195,6 +8195,16 @@ Zimpl:
   tm_scope: none
   ace_mode: text
   language_id: 411
+crontab:
+  type: data
+  aliases:
+  - cron
+  - cron table
+  filenames:
+  - crontab
+  tm_scope: text.crontab
+  ace_mode: text
```

Comment:
ACE's Tcl mode does a good job of [highlighting](https://ace.c9.io/build/kitchen-sink.html) <samp>cron(8)</samp> tables:
<img src="https://github.com/github-linguist/linguist/assets/2346707/0ec97725-bbb9-422f-9cc5-ab43742d96c7" width="853" alt="ACE's Tcl mode highlighting PR's crontab sample"/>
```suggestion
  ace_mode: tcl
```